.travis.yml

```diff
@@ -33,6 +33,8 @@ virtualenv:
 install:
   - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:$PATH
+  - git clone --depth=1 https://github.com/OCA/server-tools.git ${HOME}/server-tools
```

Comment:
It's better to add this dependency in file oca_dependencies as suggested here https://github.com/OCA/hr/pull/138#issuecomment-142921115 by @hbrunn